### ISSUE TITLE
Other Meteor subscriptions stop working

### ISSUE THREAD

**Issue by DenisGorbachev** [numtel/meteor-pg#2]
```
Meteor.publish('Avatars', ->
  liveDb.select('SELECT id, name FROM nonExistentTable')
);
```

Error is swallowed; however, that breaks other Meteor subscriptions.

P.S. Great code otherwise, congrats on your style!

**Comment by numtel** [comment]
This is fixed in version 0.0.3. Errors will be output to the console but the application will no longer hang.

Thanks for the issue!

**Comment by DenisGorbachev** [comment]
Wow, thanks for quick reply!

Updated to 0.0.3, now the errors propagate correctly. Again, thank you!
